example/image-classification/common/fit.py

```diff
@@ -305,3 +321,8 @@ def fit(args, network, data_loader, **kwargs):
               epoch_end_callback=checkpoint,
               allow_missing=True,
               monitor=monitor)
+
+    if args.profile_server_file:
+        kv.set_server_profiler_state(state='stop')
+    if args.profile_worker_file:
```

Comment:
just curious if you had any special reason to stop the server before the worker.

Reply:
Nope